plugins/transformer/basic/docs/overview.md

```diff
@@ -2,6 +2,7 @@ This CloudQuery transformer plugin provides basic transformation capabilities:
 
 - Removing columns
 - Adding literal string columns
+- Adding a column with the current timestamp
```

Comment:
Should we clarify the semantics of the value here or somewhere? i.e. that the value is the microsecond-resolution current timestamp as defined by Go's `time.Now()`, executed at the time of syncing each batch? Perhaps that's too granular, but maybe at least that two rows may have a slightly different timestamp?

Reply:
I have updated the description in [2beda7f](https://github.com/cloudquery/cloudquery/pull/19126/commits/2beda7fd5e826522c534fa6c959e9b0035fe41c2)